### PR TITLE
Improve vulnerabilities feed update

### PR DESCRIPTION
**Pull request by palaciosjeremias**
|Related issue|
|---|
|#8166|

## Description
This PR improves the vulnerabilities feed update to conserve a disk reference of the last time the feed was updated. In order to do this:
- METADATA and NVD_METADATA tables from cve.db now have a new last_update column.
- The statements to write such tables now writes the actual time in these new columns when updating the entry.
- METADATA and NVD_METADATA are now standardized to have a column named timestamp with the text value of the last update obtained from the web of the feed.
- A bug in the NVD_METADATA timestamp type column was fixed.
- A new method and statement to obtain this value from the tables are implemented. 
- NVD_METADATA has multiple rows for each feed year. The method returns the biggest value of all these rows as that last_update of NVD.
- Old **last_update** value of the structure of the feed is now renamed to **last_sync** to reflect that this value is updated on each attempt to update the feed. And a new **last_update** value was implemented to hold the value obtained from the database containing the last real update of the feed.

## Tests

- Compilation without warnings in every supported platform
  - [X] Linux
- [X] Source installation
- [X] Package installation
- [X] Source upgrade
- [X] Package upgrade

- Memory tests for Linux
  - [x] Scan-build report
  - [x] Valgrind (memcheck and descriptor leaks check)

### CODE REVIEW

**Comment by palaciosjeremias** [comment]
**Valgrind UT**:
image

**Comment by palaciosjeremias** [comment]
**Scan-build report:**
image

File: src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c [hunk 1]
```diff
@@ -174,6 +174,14 @@ STATIC int wm_vuldet_sync_feed(update_node *upd, int8_t *need_update);
 STATIC int wm_vuldet_check_feed(update_node *upd, int8_t *updated);
 STATIC vu_feed wm_vuldet_set_allowed_feed(const char *os_name, const char *os_version, update_node **updates, vu_feed *agent_dist);
 
+/**
+ * @brief Gets the last update time of an specific feed from the metadata database.
```
Comment (pereyra-m) [inline src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c:178]: Typo: an

File: src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c [hunk 2]
```diff
@@ -174,6 +174,14 @@ STATIC int wm_vuldet_sync_feed(update_node *upd, int8_t *need_update);
...
+ *
+ * @param feed The feedfrom which to obtain the information.
```
Comment (pereyra-m) [inline src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c:180]: Typo: missing space after `feed`.
Also, we can consider a different expression for this sentence.

**Review by pereyra-m — APPROVED** [review]
LGTM!

**Review by DProvinciani — APPROVED** [review]
Looks good!

**Pull request merged**